### ISSUE TITLE
YAML Load warning on Fedora

### ISSUE THREAD

**Issue by jbech-linaro** [Linaro/jipdate#31]
From Linus W:

```bash
$ ./jipdate.py -q
/home/linus/jipdate/cfg.py:103: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
 yml_config = yaml.load(yml)
```

**Comment by jbech-linaro** [comment]
Here is a bit more information about it:
https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation

**Comment by jbech-linaro** [comment]
This patch fixes the issue
```patch
diff --git a/cfg.py b/cfg.py
index adab2da..16bbfd0 100644
--- a/cfg.py
+++ b/cfg.py
@@ -100,4 +100,4 @@ def initiate_config():
 
     vprint("Using config file: %s" % config_file)
     with open(config_file, 'r') as yml:
-        yml_config = yaml.load(yml)
+        yml_config = yaml.load(yml, Loader=yaml.FullLoader)
```
However, on my first try using Debian I get:
```bash
AttributeError: module 'yaml' has no attribute 'FullLoader'
```
I.e., Debian is probably lagging behind. I'm going to try update my Python packages to see if there is any newer package.

**Comment by fboudra** [comment]
FullLoader is available from PyYAML 5.1 which many distributions don't have yet. Unless you really need FullLoader, you can use SafeLoader or the optimized versions CSafeLoader or CLoader (it requires libyaml installed because it uses the C bindings instead or pure python).
Note: most distro are building the C bindings so you're safe to use the C*Loader

**Comment by jbech-linaro** [comment]
The issue seen after providing a patch was a result of running too old PyYAML. Users need to update their PyYAML version:
```bash
$ pip3 install -U PyYAML
```

I've tested this on Debian, Ubuntu, Fedora and Arch Linux.

Patch has been pushed:
https://github.com/Linaro/jipdate/commit/87911b5320924262c993e3a05c51233a5d3e80c7
